### ISSUE TITLE
HIde green plus icon when dragging

### ISSUE THREAD

**Issue by disitec** [cameronhimself/vue-drag-drop#57]
Is there a way to disable the icon that appears when you are in the drop zone? In my app this feature is for moving elements and the feeling is that you are actually duplicating them.

I dont know if it is either macOs specific or related with brower.

Thank you!

**Comment by cameronhimself** [comment]
So, the correct way to do this is with the [`effect-allowed`](https://github.com/cameronhimself/vue-drag-drop#effect-allowed) and [`drop-effect`](https://github.com/cameronhimself/vue-drag-drop#drop-effect) properties, but I goofed and they weren't actually working. That is fixed in 1.1.4, which I just published. I've also added all of the possible cursor effects to the [demo page](https://cameronhimself.github.io/vue-drag-drop/), so take a look and see if any of those are appropriate to your use case.
